### PR TITLE
chore: add eloquent javascript e-book to resources

### LINKED ISSUE #800: 💡 [ADD] - ELOQUENT JAVASCRIPT E-BOOK TO RESOURCES
### Site URL

https://eloquentjavascript.net/

### Description of the above provided site

This is a book about JavaScript, programming, and the wonders of the digital. You can read it online here it's completely revised and updated, this best-selling introduction to programming in JavaScript focuses on writing real applications.

### Category

Resources

### Sub Category

E-Book

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Fixes Issue

- Closes #800

## Changes proposed

- Added '*Eloquent JavaScript*' e-book for resources at `e-book.js` file

## Screenshots

Screenshot 2023-05-28 at 18-59-44 LinksHub A hub of ready-to-use tech resources

### CODE REVIEW

File: database/resources/e-book.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
         "category": "resource",
         "subcategory": "e-book",
         "language": "english"
+    },
+    {
+        "name": "Eloquent JavaScript",
+        "description": "Eloquent JavaScript is all about JavaScript, programming, and the wonders of the digital. You can read it online it's completely revised and updated, one of the best-selling e-book. It focuses on writing real applications.",
```
Comment (CBID2) [inline database/resources/e-book.json:20]:
```suggestion
        "description": "Eloquent JavaScript is an e-book that describes the wonders of Javascript andd provides exercises on creating real applications.",

```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @0xabdulkhalid! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Comment by 0xabdulkhalid** [comment]
Yeah @CBID2, i have resolved the typo!

Here's the updated typo: `"Eloquent JavaScript is an e-book that describes the wonders of Javascript and provides exercises on creating real world applications."`

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by 0xabdulkhalid** [comment]
Greetings @rupali-codes, please consider to review this PR so that i can create another issue to work on it

Regards

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**